desktop/cmp/grid/GridModel.js

```diff
@@ -139,7 +139,8 @@ export class GridModel {
         enableExport = false,
         exportFilename = 'export',
         rowClassFn = null,
-        contextMenuFn = () => this.defaultContextMenu()
+        contextMenuFn = () => this.defaultContextMenu(),
+        actionMetadata = null
```

Comment:
Let's discuss the metadata issue in person -- know you were concerned about it and think it best if you explain